rules/windows/process_creation/process_creation_win_msdt_sdiagnhost.yml

```diff
@@ -0,0 +1,36 @@
+title: Sdiagnhost.exe loading Powershell Module
+id: ec8c4047-fad9-416a-8c81-0f479353d7f6
+status: experimental
+description: Detects windows document or pdf reader leveraging "msdt.exe" binary and "sdiagnohost.exe"; "sdiagnhost.exe" process loads "PowerShell.Command.Utility.dll" to execute arbitary executable command (CVE-2022-30190)
+author: Greg (rule)
+references:
+  - https://twitter.com/j00sean/status/1534115332830507008
+  - https://twitter.com/nas_bench/status/1531944240271568896?t=z0hjfsgRgNb9c4NCLk-bHg&s=19  # rule is also by proc_creation_win_msdt_sdiagnhost.yml
+  - https://conscia.com/blog/vulnerability-spotlight-how-to-detect-follina-the-windows-msdt-0-day/
+date: 2022/06/09
+modified: 2022/06/09
+logsource:
+  category: process_creation
+  product: windows
+detection:
+  selection_img:
+    - Image|endswith: '\msdt.exe'
+    - OriginalFileName: 'msdt.exe'
+  selection_cmd:
+    CommandLine|contains:
+      - 'PCWDiagnostic'
+      - '/af ' # For answer files
+  selection_load:
+    Image|endswith: '\sdiagnhost.exe'
+    ImageLoaded|contains: 'PowerShell.Command'
+    ImageLoaded|endswith: '.dll'
+
+  condition: all of selection*
```

Comment:
How can `Image` simultaneously end with `\msdt.exe` AND `\sdiagnhost.exe` ?

Did you mean `ParentImage` for `msdt.exe`?